lib/index.js

```diff
@@ -162,16 +170,28 @@ export default class HyperList {
       }
     })
 
-    // Decorate the container element with styles that will match
+    // If using React, the element object turns into a props object.
+    if (config.isReact) {
```

Comment:
Hmm, I'm not fond of those conditions. I'm sure that there is a way to manage vanillajs and react with the same code no?